index.js

```diff
@@ -5,23 +5,33 @@ const Recipe = require('./models/Recipe.model');
 // Import of the data from './data.json'
 const data = require('./data');
 
+mongoose.set('useFindAndModify', false);
+
 const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
 
 // Connection to the database "recipe-app"
 mongoose
-  .connect(MONGODB_URI, {
-    useCreateIndex: true,
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
-  .then(self => {
-    console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
-  })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
-  })
-  .catch(error => {
-    console.error('Error connecting to the database', error);
-  });
+	.connect(MONGODB_URI, {
+		useCreateIndex: true,
+		useNewUrlParser: true,
+		useUnifiedTopology: true
+	})
+	.then(self => {
+		console.log(`Connected to the database: "${self.connection.name}"`);
+		return Recipe.deleteMany()
+	})
+	.then(() => Recipe.syncIndexes())
+	.then(() => Recipe.create({title: 'Arroz', cuisine: 'international'}))
+	.then(() => {
+		Recipe.insertMany(data)
+		data.forEach(recipe => {
+			console.log(`Recipe "${recipe.title}" created`);
+		});
+	})
+	.then(() => Recipe.findOneAndUpdate({ title: 'Rigatoni alla Genovese' }, { duration: 100 }, {returnOriginal: false}))
+	.then((rigatone) => console.log(rigatone))
```

Comment:
- Para anidar promesas, el `then` tiene que hacer return de la promesa. En las enteriores te funciona porque las funciones flecha, si solo tienen una línea, hacen `return` de eso automáticamente.
- Una forma de solucionarlo y usar el `new:true` sería el siguiente:

```suggestion
      .then(() => Recipe.insertMany(data))
	.then((data) => data.forEach(recipe => console.log(`Recipe "${recipe.title}" created`)))
	.then(() => Recipe.findOneAndUpdate({ title: 'Rigatoni alla Genovese' }, { duration: 100 }, { new: true }))
	.then((rigatone) => console.log(rigatone))
```